lib/jira/util/index.js

```diff
@@ -1,6 +1,19 @@
-const { markdown } = require('markdown');
-
 module.exports = function (jiraClient) {
+  const checkForReferenceText = (text) => {
+    const splitTextByNewLine = text.split('\n');
+
+    return splitTextByNewLine
+      // reference text links should have 2 parts only
+      .filter((newLine) => newLine.split(' ').length === 2)
```

Comment:
Is the space mandatory for reference links in GitHub Flavored Markdown? Or is it possible to create such reference links without a space?